backend/src/main/java/com/woowacourse/levellog/team/domain/Team.java

```diff
@@ -93,4 +94,23 @@ public void update(final String title, final String place, final LocalDateTime s
         this.place = place;
         this.startAt = startAt;
     }
+
+    public void close(final LocalDateTime presentTime) {
+        validateInterviewStartTime(presentTime);
+        validateAlreadyClosedInterview();
+
+        isClosed = true;
+    }
+
+    private void validateInterviewStartTime(final LocalDateTime presentTime) {
+        if (presentTime.isBefore(startAt)) {
+            throw new InterviewTimeException("[teamId : " + this.getId() + "]", "인터뷰가 시작되기 전에 종료할 수 없습니다.");
+        }
+    }
+
+    private void validateAlreadyClosedInterview() {
+        if (isClosed) {
+            throw new InterviewTimeException("[teamId : " + this.getId() + "]", "이미 종료된 인터뷰입니다.");
+        }
+    }
```

Comment:
`close`메서드에서도 interview라는 단어를 뺐으니 private 메서드에서도 제거해도 좋을거 같네요!

```suggestion
    private void validateStartTime(final LocalDateTime presentTime) {
        if (presentTime.isBefore(startAt)) {
            throw new InterviewTimeException("[teamId : " + this.getId() + "]", "인터뷰가 시작되기 전에 종료할 수 없습니다.");
        }
    }

    private void validateAlreadyClosed() {
        if (isClosed) {
            throw new InterviewTimeException("[teamId : " + this.getId() + "]", "이미 종료된 인터뷰입니다.");
        }
    }
```

Reply:
시작 시간 검증 메소드는 기존의 startAt 필드를 검증하는 메소드와 이름이 헷갈릴 거 같아서 기존 이름대로 유지했어요!
이미 종료된 인터뷰 검증은 위에서 페퍼 피드백 받고 통일성 있게 고친다는 게 깜빡했네용..ㅎㅎ 수정했습니다!